pkg/stub/handler.go

```diff
@@ -526,14 +531,18 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 	case *v1alpha1.SamplesResource:
 		newStatus := corev1.ConditionTrue
 		srcfg, _ := event.Object.(*v1alpha1.SamplesResource)
+
 		if srcfg.Name != v1alpha1.SamplesResourceName || srcfg.Namespace != "" {
 			return nil
 		}
 
 		// Every time we see a change to the SamplesResource object, update the ClusterOperator status
 		// based on the current conditions of the SamplesResource.
 		err := h.cvowrapper.UpdateOperatorStatus(srcfg)
+		// TODO make sure an error here does not prevent us from doing other necessary processing
+		// of the samples resource object.
```

Comment:
So help me reconcile this comment with the fact that you have an error is nil check and returning the error to the SDK if it exists ...

... is the answer to the TODO comment not to just log the error and continue processing (and perhaps attempt to update operator status at a later date)?

If not, I need help understanding what you are getting at here ....

Reply:
it could be that simple, but i wasn't sure if we'd want to do something more sophisticated (such as forcing the event to be requeued for retry, which is obviously what currently happens with the return err).  So i wanted to revisit that as we think about the general level-driven reconciliation pattern we want to get to here.

Reply:
Do you want to reconcile this here, or in my follow up PR @bparees ?

Reply:
ah missed your last comment before posting @bparees

Reply:
For my follow up PR